doc/api/inspector.md

```diff
@@ -52,6 +52,13 @@ parameter usage.
 
 Return the URL of the active inspector, or `undefined` if there is none.
 
+## inspector.waitForDebugger()
+
+Blocks untils a client has sent `Runtime.runIfWaitingForDebugger` command using
+protocol.
```

Comment:
I believe it is worth mentioning that it would also wait for a client to connect when there's no connected clients.

Reply:
done.